R/dw.R

```diff
@@ -4,6 +4,8 @@
 #'
 #' @param param parameter naam (als string)
 #' @param gwl grondwaterlichaam (groundwaterbody)
+#' @param data dataset met drempelwaardes. default is drempelwaardes dataset die
```

Comment:
Is het dataformat van de drempelwaardedataset bekend bij de gebruiker? In de handleiding zie ik niet zo snel daar iets over staan.

Reply:
Nieuw issue aangemaakt #16. Voor het opnemen in de handleiding op een later moment.